client/src/app/gateways/http-stream/http-stream.ts

```diff
@@ -228,7 +246,7 @@ export class HttpStream<T> {
         this._shouldReconnectOnFailure = config.shouldReconnectOnFailure ?? false;
         this._reconnectTimeout = config.reconnectTimeout ?? 2000;
 
-        this.buildMessageParser();
+        this.buildMessageParser(this.endpoint.url);
```

Comment:
Why is this necessary although an endpoint can be accessed as member of an http stream?